source/unified-test-format/unified-test-format.rst

```diff
@@ -1788,6 +1820,26 @@ The following arguments are supported:
 
 - ``session``: Optional string. See `commonOptions_session`_.
 
+- ``batchSize``: Optional positive integer value. Use this value to configure the ``batchSize`` option sent on subsequent ``getMore`` commands.
+
+- ``maxTimeMS``: Optional non-negative integer value. Use this value to configure the ``maxTimeMS`` option sent on subsequent ``getMore`` commands.
+
+- ``comment``: Optional BSON value. Use this value to configure the ``comment`` option sent on subsequent ``getMore`` commands.
+
+- ``cursorType``: Optional string enum value, one of ``'tailable' | 'tailableAwait' | 'nonTailable'``. Use this value to configure the enum passed to the ``cursorType`` option.
+
+- ``timeoutMode``: Optional string enum value, one of ``'iteration' | 'cursorLifetime'``. Use this value to configure the enum passed to the ``timeoutMode`` option.
```

Comment:
Per my earlier comment about using all-caps in the RunCommand spec, I've noted that this text means we won't have any conflict with the spec tests since you're making the connection between strings and enum values here.

This is better than the CSOT spec, which doesn't talk about the mapping between its enums and the strings used in spec tests anywhere.